sui/src/validator.rs

```diff
@@ -61,14 +63,18 @@ async fn main() -> Result<(), anyhow::Error> {
         address, net_cfg.host, net_cfg.port
     );
 
+    let listen_address = cfg
+        .listen_address
+        .unwrap_or(format!("{}:{}", net_cfg.host, net_cfg.port));
+
     if let Err(e) = make_server(
         net_cfg,
         &Committee::from(&network_config),
         network_config.buffer_size,
     )
     .await
     .unwrap()
-    .spawn()
+    .spawn_with_address(&listen_address)
```

Comment:
Address is very overloaded, I'd rename to something like `spawn_with_bind_address`.